internal/generator/templates/preamble.go.tmpl

```diff
@@ -5,6 +5,8 @@ package {{ goPackageName .pkg.Name }}
 {{ $root := .rootPackage }}
 import (
 	"errors"
+    "fmt"
+    "database/sql/driver"
     "github.com/icza/bitio"
     zserio "github.com/woven-planet/go-zserio"
     "github.com/woven-planet/go-zserio/ztype"
```

Comment:
nit, the indentation is weird here. It does not matter much, because we are formatting the code afterwards anyway.

Reply:
Fixed. There was a tab used for the `errors` package.